pkg/plugins/policies/core/matchers/validators/validator.go

```diff
@@ -2,11 +2,18 @@ package validators
 
 import (
 	"fmt"
+	"regexp"
 
 	common_api "github.com/kumahq/kuma/api/common/v1alpha1"
 	"github.com/kumahq/kuma/pkg/core/validators"
 )
 
+var (
+	nameCharacterSet     = regexp.MustCompile("^[0-9a-z-_]*$")
+	tagNameCharacterSet  = regexp.MustCompile(`^[a-zA-Z0-9\.\-_:/]*$`)
+	tagValueCharacterSet = regexp.MustCompile(`^([a-zA-Z0-9\.\-_:/]*|\*)$`)
```

Comment:
I don't think you have to escape `.` and `-` inside `[]` when using backticks to describe regexp

Reply:
I removed the escape for `.` but it is required for `-` to differentiate it from the range tick. Updated accordingly 👍